pre_commit/main.py

```diff
@@ -101,6 +100,10 @@ def main(argv=None):
         help='Allow an unstaged config to be present.  Note that this will'
         'be stashed before parsing unless --no-stash is specified'
     )
+    run_parser.add_argument(
+        '--hook-stage',
```

Comment:
This'll break invocations that are just `pre-commit` and `pre-commit run`.  Maybe make it default to `'commit'`?  This might also make the bash changes easier.